src/components/entity/EntityResult.vue

```diff
@@ -277,15 +279,17 @@
           :effectiveDate="ownedByRelationship.credential.effective_date"
         >
           <template #header>
-            <router-link
-              :to="`/entity/${
-                ownedByRelationship.related_topic &&
-                ownedByRelationship.related_topic.source_id
-              }`"
-              class="font-weight-bold"
-            >
-              {{ getRelationshipName(ownedByRelationship) }}
-            </router-link>
+            <span @click="stopEvent">
```

Comment:
Take a look at: https://vuejs.org/v2/guide/events.html#Event-Modifiers and https://stackoverflow.com/questions/65389658/vue-js-prevent-event-propagation-for-router-link. I think the documentation makes sense about not handling dom events in methods but rather deal with data and application logic